---
title: "Publications"
permalink: /publications/
author_profile: true
---

{% if author.googlescholar %}
  You can also find my articles on <u><a href="{{author.googlescholar}}">my Google Scholar profile</a>.</u>
{% endif %}



## Academic publications

Mariana Matus, **Claire Duvallet**, Newsha Ghaeli, Melissa Kido Soule, Krista Longnecker, Ilana Brito, Carlo Ratti, Elizabeth B. Kujawinski, Eric Alm.   
"Untargeted detection of human health and activity markers in residential wastewater through microbiome sequencing and metabolomics."    
_bioRxiv_ (2019) doi:[10.1101/728022](https://doi.org/10.1101/728022).

**Claire Duvallet**, Caroline Zellmer, Pratik Panchal, Shrish Budree, Madji Osman, and Eric Alm.    
"Framework for rational donor selection in fecal microbiota transplant clinical trials."    
_PLoS ONE_ (2019) doi: [10.1371/journal.pone.0222881](https://doi.org/10.1371/journal.pone.0222881).     
Associated github: [cduvallet/donor-selection](https://github.com/cduvallet/donor-selection/)

Evan Bolyen, Jai Ram Rideout, Matthew R. Dillon, Nicholas A. Bokulich ... **Claire Duvallet** ... Rob Knight, Greg Caporaso.     
"Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2."      
_Nature Biotechnology_ (2019). doi: [10.1038/s41587-019-0209-9](https://doi.org/10.1038/s41587-019-0209-9).

Chengzhen Dai, **Claire Duvallet**, An Ni Zhang, Mariana Matus, Newsha Ghaeli, Shinkyu Park, Noriko Endo, Siavash Isazadeh, Kazi Jamil, Carlo Ratti, and Eric Alm.    
"Multi-site sampling and risk prioritization reveals the public health relevance of antibiotic resistance genes found in sewage environments."      
_bioRxiv_ (2019). doi: [10.1101/562496](https://doi.org/10.1101/562496).

**Claire Duvallet**, Kara Larson, Scott Snapper, Sonia Iosim, Ann Lee, Katherine Freer, Kara May, Eric Alm, and Rachel Rosen.    
"Aerodigestive sampling reveals altered microbial exchange between lung, oropharyngeal, and gastric microbiomes in children with impaired swallow function."     
_PLOS ONE_ (2019). doi: [10.1371/journal.pone.0216453](https://doi.org/10.1371/journal.pone.0216453).    
Associated github: [cduvallet/aspiration-analysis-public](https://github.com/cduvallet/aspiration-analysis-public)

Keegan Korthauer\*, Patrick K Kimes\*, **Claire Duvallet**^, Alejandro Reyes^, Ayshwarya Subramanian^, Mingxiang Teng, Chinmay Shukla, Eric J Alm, and Stephanie C Hicks.     
"A practical guide to methods controlling false discoveries in computational biology."     
_Genome Biology_ (2019). doi: [10.1186/s13059-019-1716-1](https://doi.org/10.1186/s13059-019-1716-1).    
Associated github: [pkimes/benchmark-fdr](https://github.com/pkimes/benchmark-fdr)

Thomas Gurry, HST Microbiome Consortium, Sean M Gibbons, Sean M Kearney, Ashwin Ananthakrishnan, Xiaofang Jiang, **Claire Duvallet**, Zain Kassam, and Eric J Alm.      
"Predictability and persistence of prebiotic dietary supplementation in a healthy human cohort."       
_Scientific Reports_ (2018) 8(1): 12699. doi: [10.1038/s41598-018-30783-1](https://doi.org/10.1038/s41598-018-30783-1)

Sean M. Gibbons, **Claire Duvallet**, and Eric J. Alm.      
"Correcting for batch effects in case-control microbiome studies."       
_PLoS Computational Biology_ (2018) 14(4): e1006102. doi: [10.1371/journal.pone.0176335](https://doi.org/10.1371/journal.pcbi.1006102)

**Claire Duvallet**.       
"Meta-analysis generates and prioritizes hypotheses for translational microbiome research."        
_Microbial Biotechnology_ (2018). doi: [10.1111/1751-7915.13047](https://doi.org/10.1111/1751-7915.13047)

**Claire Duvallet**, Sean M. Gibbons, Thomas Gurry, Rafael A. Irizarry, and Eric J. Alm.       
"Meta-analysis of gut microbiome studies identifies disease-specific and shared responses."        
_Nature Communications_ (2017). doi: [10.1038/s41467-017-01973-8](https://doi.org/10.1038/s41467-017-01973-8).       
Associated github: [cduvallet/microbiomeHD](https://github.com/cduvallet/microbiomeHD)     
Associated data: [zenodo record 840333](https://doi.org/10.5281/zenodo.797943).

Scott W. Olesen, **Claire Duvallet**, and Eric J. Alm.       
"dbOTU3: A new implementation of distribution-based OTU calling."      
_PloS ONE_ 12, no. 5 (2017): e0176335. doi: [10.1371/journal.pone.0176335](https://doi.org/10.1371/journal.pone.0176335)

## Non-peer reviewed academic content

_[Tutorial]_ **Claire Duvallet**. "[Updating your qiime2 plugin](https://dev.qiime2.org/latest/tutorials/updating-plugin/)." _QIIME 2 developer docs._ (2019).

_[Blog post]_ **Claire Duvallet**. "[Scientific discovery from a clinical study: surprises from the lung and stomach microbiomes.](https://go.nature.com/30rx4VZ)" _Nature Microbiology Community Forum._ May 2019.

_[Tutorial]_ **Claire Duvallet** and Mehrbod Estaki. "[QIIME 2 for Experienced Microbiome Researchers](https://docs.qiime2.org/2018.11/tutorials/qiime2-for-experienced-microbiome-researchers/)." _QIIME 2 docs._ (2018).

_[Tutorial]_ **Claire Duvallet**. "[Developing a plug-in for dummies](https://dev.qiime2.org/latest/tutorials/first-plugin-tutorial/)." _QIIME 2 developer docs._ (2018).

_[Tutorial]_ **Claire Duvallet**. "[Publishing your plugin on conda](https://dev.qiime2.org/latest/tutorials/conda-tutorial/)." _QIIME 2 developer docs._ (2018).

_[Blog post]_ **Claire Duvallet**. "[Fuzzy zeros in percentile normalization method to correct for batch effects](https://microbe.net/2018/06/04/fuzzy-zeros-in-percentile-normalization-method-to-correct-for-batch-effects/)." _microBEnet: the microbiology of the Built Environment network._ June 2018.

_[Blog post]_ **Claire Duvallet**. "[Beyond dysbiosis: disease-specific and shared microbiome responses to disease.](http://go.nature.com/2As9meL)" _Nature Microbiology Community Forum._ December 2017.

_[Dataset]_ **Claire Duvallet**, Sean M. Gibbons, Thomas Gurry, Rafael A. Irizarry, and Eric J. Alm. (2017). MicrobiomeHD: the human gut microbiome in health and disease. _Zenodo_. doi: [10.5281/zenodo.797943](https://doi.org/10.5281/zenodo.797943)

## Software

**Percentile normalization**     
Correcting batch effects in case-control microbiome studies. (_Gibbons et al., 2018_)

* Python implementation: [seangibbons/percentile_normalization](https://github.com/seangibbons/percentile_normalization) _(contributor)_     
* [QIIME 2](https://qiime2.org/) plugin: [cduvallet/q2-perc-norm](https://github.com/cduvallet/q2-perc-norm) _(developer)_

**Distribution-based OTU calling**    
New implementation of Preheim _et al._'s distribution-based OTU clustering algorithm. (_Preheim et al., 2013_; _Olesen et al., 2017_).

* Python implementation: [swo/dbotu3](https://github.com/swo/dbotu3) _(contributor)_
* [QIIME 2](https://qiime2.org/) plugin: [cduvallet/q2-dbotu](https://github.com/cduvallet/q2-dbotu) _(developer)_

**Amplicon sequencing pipeline**      
End-to-end pipeline to process 16S data.     

* Python: [thomasgurry/amplicon_sequencing_pipeline](https://github.com/thomasgurry/amplicon_sequencing_pipeline) _(co-developer)_

**Match HMDB**     
Script to match mz's to the HMDB database.

* Python: [cduvallet/match_hmdb](https://github.com/cduvallet/match_hmdb) _(developer)_

## Other

_[Blog post]_ **Claire Duvallet**. "[Learning to Engage in Deep Conversations](https://gradadmissions.mit.edu/blog/learning-engage-deep-conversations)." _MIT Graduate Admissions blog_ (2018).

_[Resource]_ **Claire Duvallet** and Monika Avello. "[Graduate student support resources flowchart.](/files/2018-04-01-Grad-Support-Flowchart-MIT-Digital.pdf )" I and another student on [iREFS](http://refs.mit.edu/irefs/) created this flowchart to help graduate students find support at MIT.
